amoro-ams/src/main/java/org/apache/amoro/server/persistence/mapper/TableMetaMapper.java

```diff
@@ -244,7 +257,7 @@ TableMetadata selectTableMetaByName(
   @Insert(
       "INSERT INTO table_identifier(catalog_name, db_name, table_name, format) VALUES("
           + " #{tableIdentifier.catalog}, #{tableIdentifier.database}, #{tableIdentifier.tableName}, "
-          + " #{tableIdentifier.format})")
+          + " #{tableIdentifier.format, typeHandler=org.apache.amoro.server.persistence.converter.TableFormatConverter})")
```

Comment:
It seems the full class name is not necessary here.

Reply:
Only fullname can be used here